Builds/containers/packaging/dpkg/debian/rules

```diff
@@ -16,31 +16,36 @@ override_dh_systemd_start:
 
 override_dh_auto_configure:
 	env
-	rm -rf bld && mkdir -p bld/rippled
-	cd bld/rippled && \
-	cmake ../.. -G Ninja \
-		-DCMAKE_INSTALL_PREFIX=/opt/ripple \
+	rm -rf bld
+
+	conan export external/snappy snappy/1.1.9@
+
+	conan install . -if bld/rippled  --build missing --settings build_type=Release
+	cmake -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake \
+		-G Ninja \
 		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=/opt/ripple \
 		-Dstatic=ON \
 		-Dunity=OFF \
+		-DCMAKE_VERBOSE_MAKEFILE=ON \
 		-Dvalidator_keys=ON \
-		-Dunity=OFF \
-		-DCMAKE_VERBOSE_MAKEFILE=OFF
-
+		-B bld/rippled
 
-	cmake -S . \
-		-B bld/rippled-reporting \
+	conan install . -if bld/rippled-reporting --build missing --settings build_type=Release --options reporting=True -s compiler.cppstd=17
```

Comment:
```suggestion
	conan install . --install-folder bld/rippled-reporting --build missing --settings build_type=Release --options reporting=True --options cassandra-cpp-driver:use_atomic=std
```

C++17 might have worked, but I don't like that solution. Did you try the other option I suggested?